portfolio/src/main/webapp/index.html

```diff
@@ -8,9 +8,31 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Charlie Floeder</h1>
+      <h2>Quick Bio</h2>
+      <p>
+            I was born March 4, 2001 in Minnesota. I am the 7th of 10 kids in my family. 
+            There are 7 boys and 3 girls. Because of that, there was always things going on at my house. While there may 
```

Comment:
There is some line wrapping going on in the diff viewer here. Can you update this to keep line length <= 80 chars? This will make this a little easier to review/reread in the future.

If changing this breaks how this page renders, feel free to ACK this comment.